bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetAutomationHandler.java

```diff
@@ -0,0 +1,427 @@
+        }
+        updateState(CHANNEL_SHUTTER, UnDefType.UNDEF);
+        positionEstimation = POSITION_UNKNOWN;
+    }
+
+    @Override
+    protected void requestChannelState(ChannelUID channel) {
+        logger.debug("requestChannelState() thingUID={} channel={}", thing.getUID(), channel.getId());
+        Where w = deviceWhere;
+        if (w != null) {
+            try {
+                send(Automation.requestStatus(w.value()));
+            } catch (OWNException e) {
+                logger.warn("Exception while requesting channel {} state: {}", channel, e.getMessage());
+            }
+        }
+    }
+
+    @Override
+    protected void handleChannelCommand(ChannelUID channel, Command command) {
+        switch (channel.getId()) {
+            case CHANNEL_SHUTTER:
+                handleShutterCommand(command);
+                break;
+            default: {
+                logger.warn("Unsupported channel UID {}", channel);
+            }
+        }
+    }
+
+    /**
+     * Handles Automation Roller shutter command (UP/DOWN, STOP/MOVE, PERCENT xx%)
+     */
+    private void handleShutterCommand(Command command) {
+        Where w = deviceWhere;
+        if (w != null) {
+            calibrating = CALIBRATION_INACTIVE; // cancel calibration if we receive a command
+            commandRequestedWhileMoving = null;
+            try {
+                if (StopMoveType.STOP.equals(command)) {
+                    send(Automation.requestStop(w.value()));
+                } else if (command instanceof UpDownType || command instanceof PercentType) {
+                    if (movingState == MOVING_STATE_MOVING_UP || movingState == MOVING_STATE_MOVING_DOWN) { // already
+                                                                                                            // moving
+                        logger.debug("# {} # already moving, STOP then defer command", deviceWhere);
+                        commandRequestedWhileMoving = command;
+                        sendHighPriority(Automation.requestStop(w.value()));
+                        return;
+                    } else {
+                        if (command instanceof UpDownType) {
+                            if (UpDownType.UP.equals(command)) {
+                                send(Automation.requestMoveUp(w.value()));
+                            } else {
+                                send(Automation.requestMoveDown(w.value()));
+                            }
+                        } else if (command instanceof PercentType) {
+                            handlePercentCommand((PercentType) command, w.value());
+                        }
+                    }
+                } else {
+                    logger.warn("Unsupported command {} for thing {}", command, thing.getUID());
+                }
+            } catch (OWNException e) {
+                logger.warn("Exception while sending request for command {}: {}", command, e.getMessage());
```

Comment:
If this would indicate a problem with the binding you should also log the stacktrace:
```suggestion
                logger.warn("Exception while sending request for command {}: {}", command, e.getMessage(), e);
```
Also applies to other locations this similar log warning is done.

Reply:
It may happen in case of communications problems with the gateway, so I lowered it to INFO